src/Pages/Login/Login.js

```diff
@@ -1,6 +1,205 @@
+      [e.target.name]: e.target.value,
+    };
+    setInputValue(nextInputValue);
+    console.log('inputValue: ', inputValue);
+    console.log('idValue: ', idValue);
+  };
+
+  //master
+  //id:soojsooj
+  //pw:PW1!soojsooj
+
+  //seller
+  // id: seller1
+  //pw: PW1!seller1
+  const goToHome = (e) => {
+    e.preventDefault();
+    fetch('http://10.251.1.180:5000/account/signin', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        identification: idValue,
+        password: pwValue,
+      }),
+    })
+      .then((response) => response.json())
+      // .then((result) => console.log(result));
+      // .then((response) => {
+      //   if (response.token) {
+      //     localStorage.setItem('wtw-token', response.token);
+      //   }
+      // });
+      // .then((result) => console.log('result: ', result));
+      .then((result) => {
+        if (
+          result.Authorization &&
+          idValue.length >= 5 &&
+          pwValue.length >= 5
+        ) {
+          thislocalStorage.setItem('token', result.Authorization);
+          alert('로그인 성공');
+          // history.push('./');
+        } else if (result.message === 'UNAUTHORIZED') {
+          alert('아이디와 비밀번호를 확인해주세요');
+        }
+      });
+  };
+
+  return (
+    <Container>
+      <Content>
+        <Logo alt='브랜디로고' src='/public/Images/logo2.png' />
+        <LoginBox>
+          <LoginTitle>브랜디 어드민 로그인</LoginTitle>
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <Input
+              name='idValue'
+              ref={register({ required: true })}
+              placeholder='셀러 아이디'
+              className={errors.idValue && 'ErrorInput'}
+              onChange={handleInput}
+            />
+            {/* id와 password가 입력되지 않았을 때 나타날 오류 */}
+            {errors.idValue && errors.pwValue && <p>아이디를 입력해주세요</p>}
+            <Input
+              name='pwValue'
+              // ref={register({ required: true, pattern: /^[A-Za-z]+$/i })}
+              ref={register({ required: true })}
+              placeholder='셀러 비밀번호'
+              className={errors.pwValue && 'ErrorInput'}
+              onChange={handleInput}
+            />
+            {/* id와 password가 입력되지 않았을 때 나타날 오류 */}
+            {errors.pwValue && <p>비밀번호를 입력해주세요</p>}
+
+            <Button onClick={goToHome}>로그인</Button>
+            {/* <Button>로그인</Button> */}
+            <Join>
+              <p>아직 셀러가 아니신가요?</p>
+              <p>
+                <Link to='/Signup'>회원가입하기</Link>
+              </p>
+            </Join>
+          </form>
+        </LoginBox>
+      </Content>
+      <LoginFooter />
+    </Container>
+  );
 }
+
+const Container = styled.div`
+  height: 100vh;
+  background-color: #fafafa;
+`;
+
+const Content = styled.div`
+  ${({ theme }) => theme.flex('center', 'center', 'column')}
+  height: 100vh;
+  padding: 65px 0 50px;
+  p {
+    margin-top: 5px;
+    font-size: 12px;
```

Comment:
```suggestion
  padding: 65px 0 50px;
  
  p {
    margin-top: 5px;
    font-size: 12px;
```
nesting 할 때에 convention도 한줄 띄우면 가독성이 더 좋을 것 같아요~!!